JitsiParticipant.js

```diff
@@ -33,7 +33,7 @@ export default class JitsiParticipant {
             video: undefined
         };
         this._hidden = hidden;
-        this._isConnectionActive = true;
+        this._connectionStatus = null;
```

Comment:
We may want to start from ACTIVE until anything happens (lastN or any other event happens). WDYT ?